library/src/hcc_detail/hipsparse.cpp

```diff
@@ -1823,6 +1823,172 @@ hipsparseStatus_t hipsparseZbsrmv(hipsparseHandle_t         handle,
                          (rocsparse_double_complex*)y));
 }
 
+hipsparseStatus_t hipsparseSbsrmm(hipsparseHandle_t         handle,
+                                  hipsparseDirection_t      dirA,
+                                  hipsparseOperation_t      transA,
+                                  hipsparseOperation_t      transB,
+                                  int                       mb,
+                                  int                       n,
+                                  int                       kb,
+                                  int                       nnzb,
+                                  const float*              alpha,
+                                  const hipsparseMatDescr_t descrA,
+                                  const float*              bsrValA,
+                                  const int*                bsrRowPtrA,
+                                  const int*                bsrColIndA,
+                                  int                       blockDim,
+                                  const float*              B,
+                                  int                       ldb,
+                                  const float*              beta,
+                                  float*                    C,
+                                  int                       ldc)
+{
+    return rocSPARSEStatusToHIPStatus(rocsparse_sbsrmm((rocsparse_handle)handle,
+                                                       (rocsparse_direction)dirA,
```

Comment:
should be transformed by `hipDirectionToHCCDirection()` (also for other precisions)

Reply:
Fixed in latest push